### PR TITLE
water use wording changes for clarity

### PR DESCRIPTION
**Pull request by mhines-usgs**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
https://github.com/usgs-makerspace/makerspace-sandbox/issues/679

Description
-----------
Updating language based on feedback from reviewer

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/assets/questionsAndAnswers/questionsAndAnswers.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ export default {
                     {
                         "id": "useQuestion1",
                         "question": "What does this map show me?",
-                        "answer": "The maps show seasonal-average daily average water withdrawals during 2015 divided into 3 month periods for thermoelectric, irrigation, and public supply categories of use within each watershed (<a\n" +
```
Comment (jenniferRapp) [inline src/assets/questionsAndAnswers/questionsAndAnswers.js:178]: We should remove one "average" from this phrase.  
"the maps show seasonal-average daily water withdrawals...."  seems to work for me.

Comment (jenniferRapp) [reply src/assets/questionsAndAnswers/questionsAndAnswers.js:178]: I like the red better, which I believe is the old text.

File: src/assets/questionsAndAnswers/questionsAndAnswers.js [hunk 2]
```diff
@@ -243,7 +243,7 @@ export default {
                 "questionsAndAnswers": [
                     {
                         "id": "useDeepScience1",
-                        "question": "Where does the National Water Use map get the data it is presenting?",
+                        "question": "What is the source of the data presented in the National Water Use Map?",
```
Comment (jenniferRapp) [inline src/assets/questionsAndAnswers/questionsAndAnswers.js:246]: looks great

**Review by jenniferRapp — CHANGES_REQUESTED** [review]

File: src/assets/questionsAndAnswers/questionsAndAnswers.js (hunk 1 above)
Comment (mhines-usgs) [reply src/assets/questionsAndAnswers/questionsAndAnswers.js:178]: ok, i put the original text back with your suggested edit! thanks!

**Pull request merged**